src/components/MainNavBar/MainNavBar.scss

```diff
@@ -56,6 +56,14 @@ html {
         text-decoration: underline;
       }
     }
+    .fulllogo {
```

Comment:
Doesn’t look like this is being used anywhere.

Reply:
removed